changelog/1392.doc.rst

```diff
@@ -0,0 +1 @@
+Adding Hollweg Dispersion notebook to docs
```

Comment:
```suggestion
Added the Hollweg dispersion notebook.
```

Just a little rephrasing to be consistent with [PlasmaPy's changelog guidelines](https://docs.plasmapy.org/en/latest/contributing/changelog_guide.html#changelog-guidelines).